src/lib/core.js

```diff
@@ -7627,24 +7635,29 @@ export default function (context, pluginCallButtons, plugins, lang, options, _re
             pluginCallButtons = newToolbar.pluginCallButtons;
             let plugin, button, oldButton;
             for (let key in pluginCallButtons) {
-                if (!util.hasOwn(pluginCallButtons, key)) continue;
-                plugin = plugins[key];
-                button = pluginCallButtons[key];
-                if (plugin.active && button) {
-                    oldButton = oldCallButtons[key];
-                    core.callPlugin(key, null, oldButton || button);
-                    if (oldButton) {
-                        button.parentElement.replaceChild(oldButton, button);
-                        pluginCallButtons[key] = oldButton;
+                if (util.hasOwn(pluginCallButtons, key)) {
+                    plugin = plugins[key];
+                    button = pluginCallButtons[key];
+                    if (plugin.active && button) {
+                        oldButton = oldCallButtons[key];
+                        core.callPlugin(key, null, oldButton || button);
+                        if (oldButton) {
+                            button.parentElement.replaceChild(oldButton, button);
+                            pluginCallButtons[key] = oldButton;
+                        }
                     }
                 }
+                
+                if (context[key]) context[key].targetButton = pluginCallButtons[key];

```

Comment:
bug-fix with obsolete custom buttons in context

Reply:
There is no need in this bug fix if you will apply the changes from https://github.com/JiHong88/SunEditor/pull/839 pull-request